src/redturtle/volto/config/__init__.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+
+MAX_LIMIT = int(os.environ.get("REDTURTLE_VOLTO_MAX_LIMIT_SEARCH", None) or 500)
```

Comment:
shouldn't be better _os.environ.get("REDTURTLE_VOLTO_MAX_LIMIT_SEARCH", 500) or 500_ ?

Reply:
Ah, my mistake.

Reply:
> shouldn't be better _os.environ.get("REDTURTLE_VOLTO_MAX_LIMIT_SEARCH", 500) or 500_ ?

int(os.environ.get("REDTURTLE_VOLTO_MAX_LIMIT_SEARCH") or 500)

Reply:
Done